### PR TITLE
remove text example from tests (2nd option for fixing #222)

### PR DESCRIPTION
**Pull request by shashi**

### CODE REVIEW

**Comment by shashi** [comment]
what should I do about the nightly test fails? @tkelman

File: test/compare_examples.jl [hunk 1]
```diff
@@ -2,7 +2,20 @@
 cachedout = joinpath(testdir, "data")
 differentfiles = AbstractString[]
 const creator_producer = r"(Creator|Producer)"
-for output in readdir(cachedout)
+
+const KERNEL = VERSION < v"0.5.0-dev" ? OS_NAME : Sys.KERNEL
```
Comment (tkelman) [inline test/compare_examples.jl]: use Compat, this cutoff isn't right

**Comment by tkelman** [comment]
looks like https://github.com/JuliaLang/julia/pull/18839 may have broken precompilation of Iterators.jl. I suspected we shouldn't export the Base.Iterators module. Reduce and report to base, or see if removing `Iterators` from `base/exports.jl` is enough to fix it locally

**Comment by tlnagy** [comment]
Any progress on this? Gadfly is also having a lot of warnings due to JuliaLang/julia#18839 and I'm not sure how to patch everything given the reliance on both Iterators and Base.Iterators.

**Pull request closed without merging**